examples/plot_digits.py

```diff
@@ -60,7 +64,7 @@
 # Create the visualizations (increased the graph_gravity for a tighter graph-look.)
 print("Output graph examples to html" )
 # Tooltips with image data for every cluster member
-mapper.visualize(graph,
+vis_1 = mapper.visualize(graph,
```

Comment:
You mind running `black` on these examples? Looks like the formatting has drifted.

Reply:
Done, and will do for future changes too